includes/connect/class-wc-stripe-connect-api.php

```diff
@@ -0,0 +1,244 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! defined( 'WOOCOMMERCE_CONNECT_SERVER_URL' ) ) {
+	define( 'WOOCOMMERCE_CONNECT_SERVER_URL', 'https://api.woocommerce.com/' );
+}
+
+if ( ! class_exists( 'WC_Stripe_Connect_API' ) ) {
+	/**
+	 * Stripe Connect API class.
+	 */
+	class WC_Stripe_Connect_API {
+
+		const WOOCOMMERCE_CONNECT_SERVER_API_VERSION = '3';
+
+		/**
+		 * Send GET request for Stripe account details
+		 *
+		 * @return array
+		 */
+		public function get_stripe_account_details() {
+			return $this->request( 'GET', '/stripe/account' );
+		}
+
+		/**
+		 * Send request to Connect Server to initiate Stripe OAuth
+		 *
+		 * @param  string $return_url return address.
+		 *
+		 * @return array
+		 */
+		public function get_stripe_oauth_init( $return_url ) {
+
+			$current_user                   = wp_get_current_user();
+			$business_data                  = array();
+			$business_data['url']           = get_site_url();
+			$business_data['business_name'] = html_entity_decode( get_bloginfo( 'name' ), ENT_QUOTES );
+			$business_data['first_name']    = $current_user->user_firstname;
+			$business_data['last_name']     = $current_user->user_lastname;
+			$business_data['phone']         = '';
```

Comment:
WooCommerce core, which is likely active at this point, may have a phone number for the current user in user meta.